gravitee-apim-portal-webui-next/src/scss/theme/theme.scss

```diff
@@ -58,6 +58,7 @@ $theme-typography: mat.define-rem-typography-config();
 
 // Variables
 $banner-background-color: #eaddff;
+$button-selected-state-background-color: #eaddff;
```

Comment:
In thinking of the theme, seeing as this color is the same as the banner, Audrey should specify if those two items can be configured separately. 

Additionally, this color `#eaddff` is used also for the avatar. Audrey needs to also specify if the color of the filters should match the avatar color.